### PR TITLE
feat(ci): dispatch bump-sentry

### PR DESCRIPTION
**Pull request by joshuarli**
Getsentry piece (current bin/bump-sentry) which needs to be merged first: https://github.com/getsentry/getsentry/pull/7706

Also tested to work with the new bump sentry from https://github.com/getsentry/sentry/pull/34879 and https://github.com/getsentry/getsentry/pull/7587, as simulated in getsentry-test-repo.

https://github.com/getsentry/sentry/actions/runs/2552218539 triggers https://github.com/getsentry/getsentry-test-repo/actions/runs/2552238467 which bumped sentry:

```
[master df966cf] getsentry/sentry@a757acf6ea00da514bb36aff93576aaba9e0a199
 6 files changed, 6 insertions(+), 6 deletions(-)
To https://github.com/getsentry/getsentry-test-repo
   2dfb7e5..df966cf  master -> master
```

https://github.com/getsentry/getsentry-test-repo/commit/df966cf30381a1ce5407df02040601960d3d28fa

### CODE REVIEW

**Comment by joshuarli** [comment]
yay https://github.com/getsentry/getsentry-test-repo/runs/7032436456?check_suite_focus=true

File: .github/workflows/dispatch-bump-sentry.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    branches:
+      - master
+
+jobs:
+  dispatch:
+    name: dispatch
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Dispatch getsentry tests
+        uses: actions/github-script@v6
```
Comment (armenzg) [inline .github/workflows/dispatch-bump-sentry.yml:14]: Should we pin this to a sha?

**Review by asottile-sentry — APPROVED** [review]

File: .github/workflows/dispatch-bump-sentry.yml (hunk 1 above)
Comment (joshuarli) [reply .github/workflows/dispatch-bump-sentry.yml:14]: deferring this to when i have some time to get to https://getsentry.atlassian.net/jira/software/c/projects/DEVINFRA/boards/172?modal=detail&selectedIssue=DEVINFRA-38 where i'll just pin everyhting

**Review by asottile-sentry — APPROVED** [review]

**Pull request merged**